flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -553,9 +553,9 @@ function kill_all {
 }
 
 function kill_random_taskmanager {
```

Comment:
Does this change also other tests? If yes, I would move it to a separate commit.